docs/Install.md

```diff
@@ -7,7 +7,7 @@
 If you want to customize the installation of the Splunk Operator, download a copy of the installation YAML locally, and open it in your favorite editor.
 
 ```
-wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.2/splunk-operator-install.yaml
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.3/splunk-operator-install.yaml
```

Comment:
Aren't there other places where the 1.0.2 to 1.0.3 need to be changed? Check L21 below. Here is the helper [PR](https://github.com/splunk/splunk-operator/pull/315/files) to develop for 1.0.0GA release. I would recommend doing an apples to apples comparison once. It's easy to miss semantic changes I guess.